value-no-numeric-constants.js

```diff
@@ -86,6 +95,10 @@ const rule = stylelint.createPlugin(ruleName, function(primaryOption) {
 				return;
 			}
 
+			if (isExistAllowRange && + decl.value < allowLt && + decl.value > allowGt) {
```

Comment:
"lt" and "gt" are exclusive, and that's not really what developers typically expect.

Say, "z-index should be in range -10 to 10" becomes `{allowLt: 11, allowGt: -11}`.

I'd really prefer it like this: `{min: 10, max: 10}`

Reply:
Or, 
```
{lt: -10, gt: 10}
```
Like, "disallow numeric constants lower that -10 and greater than 10"